### DISCUSSION TITLE
[Feature] Make the delete button more accessible on web

### DISCUSSION THREAD

**Discussion by YarosMallorca** [immich-app/immich#14632] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Something that I think we all do pretty often, is look through the recent photos and delete unnecessary things, and using the WebUI, it's kind of annoying that the button is buried in a submenu.

My suggestion is to replace the "select all" button with the delete button, so:
- place the delete button next to the three-dot menu
- put the select all button into the submenu, because it's not something I imagine is widely-used

Similarly to how it is in google photos:
<img src="https://github.com/user-attachments/assets/0df9b92d-96e4-426b-bef2-3b266533fa54" height="70px" />

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by neohemp** [comment]
upvoted
